packages/ui-admin/src/workspace/bots/CreateBotModal.tsx

```diff
@@ -1,6 +1,6 @@
 import { Button, Classes, Dialog, FormGroup, InputGroup, Intent, Callout } from '@blueprintjs/core'
 import { BotConfig, BotTemplate } from 'botpress/sdk'
-import { lang } from 'botpress/shared'
+import { Checkbox, lang } from 'botpress/shared'
```

Comment:
Can you take the standard one from blueprint instead ? I'm not sure we'll use those components anymore